src/Host.cpp

```diff
@@ -409,7 +409,7 @@ void Host::startSpeedWalk()
     int totalWeight = assemblePath();
     Q_UNUSED(totalWeight);
     QString f = QStringLiteral("doSpeedWalk");
-    QString n = QStringLiteral("");
+    QString n = QString();
     mLuaInterpreter.call( f, n );
```

Comment:
Go the whole hog, if we have tweaked it to be `TLuaInterpreter::call(const QString &, const QString &)` by now you ought to be able to get rid of `f` and `n` anyway:

    mLuaInterpreter.call(QStringLiteral("doSpeedWalk"), QString());